vignettes/adbcva.Rmd

```diff
@@ -268,33 +268,42 @@ dataset_vignette(
 ```
 ## Deriving Criterion Flags for BCVA Change {#critflags}
 
-`{admiralophtha}` suggests the use of criterion flag variable pairs (`CRITx`/`CRITxFL`) to program BCVA endpoints such as *Avoiding a loss of x letters*  or *Gain of y letters* or *Gain of between x and y letters* (relative to baseline or other basetypes). The package provides the function `derive_var_bcvacritxfl()` to program these endpoints efficiently and consistently. In terms of the logic to apply to the variable `CHG`, the endpoints fall into three classes, which can be represented by inequalities:
+`{admiralophtha}` suggests the use of criterion flag variable pairs (`CRITx`/`CRITxFL`) to program BCVA endpoints such as *Avoiding a loss of x letters*  or *Gain of y letters* or *Gain of between x and y letters* (relative to baseline or other basetypes). The function `derive_vars_crit_flag()` can be used to derive all these criterion pairs by listing the criterion number, condition such as:
 
-* Class 1: `CHG` value lying inside a range, `a <= CHG <= b`.
-* Class 2: `CHG` value below an upper limit, `CHG <= a`.
-* Class 3: `CHG` value above a lower limit, `CHG => b`.
+* `CHG` value lying inside a range, `CHG >= a & CHG <= b`
+* `CHG` value below an upper limit, `CHG <= a`
+* `CHG` value above a lower limit, `CHG => b`
 
-By using `derive_var_bcvacritxfl()`, the ADaM programmer can implement all three types of endpoint at once. This is achieved by feeding the appropriate ranges, upper limits and lower limits to the `bcva_ranges`, `bcva_uplims` and `bcva_lowlims` arguments of the function. For instance, let's suppose that the endpoints of interest are:
+and their corresponding description of the criterion. The critical flag `CRITxFL` is generated if `values_yn` is set to TRUE.
```

Comment:
```suggestion
and their corresponding description of the criterion. The criterion flag `CRITxFL` is generated if `values_yn` is set to TRUE.
```
Second sentence needs a bit more clarity. For reference, the description of the argument `values_yn` reads: 
_Should "Y" and "N" be used for CRITyFL?If set to TRUE, the CRITyFL variable is set to "Y" if the condition (condition) evaluates to TRUE, it is set to "N" if the condition evaluate to FALSE, and to NA if it evaluates to NA.Otherwise, the CRITyFL variable is set to "Y" if the condition (condition) evaluates to TRUE, and to NA otherwise._